create_test_data.py

```diff
@@ -0,0 +1,47 @@
+import json
```

Comment:
Changed name from `create_users.py` to `create_test_data.py` , git is treating it as deleted and added file for some reason